src/java/picard/analysis/CollectMultipleMetrics.java

```diff
@@ -255,6 +255,9 @@ public SinglePassSamProgram makeInstance(final String outbase, final File input,
     @Option(shortName="LEVEL", doc="The level(s) at which to accumulate metrics.")
     public Set<MetricAccumulationLevel> METRIC_ACCUMULATION_LEVEL = new HashSet<MetricAccumulationLevel>(accumLevelDefault);
 
+    @Option(shortName = "EXT", doc="The file extension. None if null", optional=true)
```

Comment:
Would be nice to explain the use of this a bit more - that the extension will be added to all filenames generated, not replace existing extensions.  I.e. you get OUTPUT.insert_size_metrics.EXT, not OUTPUT.EXT.